platforms/cpus/gemini.repl

```diff
@@ -0,0 +1,27 @@
+//Gemini SoC platform
+
+//Memory
+ddr: Memory.MappedMemory @ sysbus 0x0
+    size: 0x80000000
+
+sram: Memory.MappedMemory @ sysbus 0x80000000
+    size: 0x80000
```

Comment:
Isnt size 256KB divided in 4 banks of 64 KB each starting from 0x80000000, 0x80010000, 0x80020000, 0x80030000?

Reply:
Thanks for the observation. I overlooked this value. Changed accordingly.